test/helpers/FormattersSpec.scala

```diff
@@ -111,4 +113,19 @@ class FormattersSpec extends SpecBase {
     }
   }
 
+  "date formatter" should {
```

Comment:
This description is not correct, there is nothing related to this term even the file id Formatter not date formatter.
It should be replaced with relevant method name like dateAsMonth or dateAsMonthAndYear

Then keep only test related to that method. It should be like below

  "dateAsMonth" should {
    "return month as a valid value" in new Setup {
      dateAsMonth(date) mustBe "July"
    }
  }

  "dateAsMonthAndYear" should {
    "return month and year as a valid value" in new Setup {
      dateAsMonthAndYear(date) mustBe "July 2020"
    }
  }